test/harvest.test.js

```diff
@@ -0,0 +1,24 @@
+import fetchMock from 'fetch-mock';
+
+import { API_BASE_URL, getHarvestJSON } from '../src/lib/harvest';
+
+describe('harvest', () => {
+  test('gets JSON data from Harvest via XHR', async () => {
+    const trello = {
+      get: (scope, visibility, key) => `${scope + visibility + key}TestValue`,
+      loadSecret: (key) => `${key}TestSecret`,
+    };
+    const url = `${API_BASE_URL}some/path`;
+    fetchMock.get(url, { some: 'data' });
```

Comment:
I generally prefer `getOnce` to be more explicit.

```suggestion
    fetchMock.getOnce(url, { some: 'data' });
```